src/pay/types.ts

```diff
@@ -0,0 +1,21 @@
+export type HydrateChargeAPIResponse = {
+  id: string;
+  callData: {
+    deadline: string;
+    feeAmount: string;
+    id: string;
+    operator: `0x${string}`;
```

Comment:
When we can, let's use the type from viem.

In this case
```
import type { Address } from 'viem';
````

Reply:
Updated